src/cascadia/WindowsTerminal/pch.h

```diff
@@ -58,6 +58,7 @@ Module Name:
 // Additional headers for various xaml features. We need:
 //  * Controls for grid
 //  * Media for ScaleTransform
+#include <winrt/Windows.UI.Core.h>

```

Comment:
huh, this needs WUC?

Reply:
Yea, the dispatcher call does. Otherwise it doesn't know how to `resume_foreground`. Yes, seriously, it's mental.